modules/ROOT/pages/io-connectors.adoc

```diff
@@ -0,0 +1,110 @@
+= Luna Streaming IO Connectors
+
+When you have Luna Streaming installed and running, add IO connectors to your deployment to interact with external systems, such as Apache Cassandra, Elastic Search, and many others.
```

Comment:
```suggestion
When you have Luna Streaming installed and running, add IO connectors to your deployment to interact with external systems, such as Apache Cassandra®, Elasticsearch, and others.
```

@mendonk Make sure you use ® for first use of Apache Cassandra on each page and format Elasticsearch properly each time (no space, lowercase S).